### ISSUE TITLE
[NVDA + Chrome] ARIA listitem role is not read

### ISSUE THREAD

**Issue by harshvardhan3946** [nvaccess/nvda#10285]
### Steps to reproduce:
Place below html on page and try reading it with nvda
<section role="list">
  <div role="listitem">List item 1</div>
  <div role="listitem">List item 2</div>
  <div role="listitem">List item 3</div>
</section>

Above code snippet is taken from
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Listitem_role
### Actual behavior:
NVDA screen reader is reading contents and not announcing list item position in list.

### Expected behavior:
NVDA should read List item 1 of 3 and recognize position and total count.

### System configuration
Windows 10, Chrome

#### Name and version of other software in use when reproducing the issue: 
Chrome Version 76.0.3809.132 (Official Build) (64-bit)

#### Does the issue still occur after restarting your PC?
YES

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by harshvardhan3946** [comment]
Please use pen https://codepen.io/harshvardhan3946/pen/dybLeKX

**Comment by sandeepnmenon** [comment]
+1 
Facing similar issue only in NVDA

**Comment by stefaniehansen** [comment]
+1 Facing same issue in NVDA 2019.2.1 and 2019.3.1 and Chrome 80.0.3987.163 and Chrome 81.0.4044.92

**Comment by Adriani90** [comment]
cc: @feerrenrut, @michaelDCurran I think there was something similar in the past which needed an external fix from Crome. I am not sure if this was the issue. Maybe you could have a look.

**Comment by feerrenrut** [comment]
@harshvardhan3946 @sandeepnmenon @stefaniehansen Can you report on the behavior with Firefox please?

**Comment by feerrenrut** [comment]
Actually looking into this more, I question that these should be read as ordered items at all. From my testing, in both chrome and firefox the sample is announced as "List with 3 items". They are announced in the same way that `<ul>` elements are. See https://www.scottohara.me/blog/2018/05/26/aria-lists.html

Is there something that makes you expect otherwise?

**Comment by Adriani90** [comment]
Ok I think I can reproduce this issue.
I think the best solution is to implement a way to announce item position and total number of items when navigating with i in browse mode from item to item (i.e. list item 1, 1 of 3) and in focus mode when pressing tab if the list contains focusable elements like i.e. in this example:

```
<section role="list">
  <div role="listitem"><button>List item 1</button></div>
  <div role="listitem"><button>List item 2</button></div>
  <div role="listitem"><button>List item 3</button></div>
</section>
```

**Comment by dballesteros7** [comment]
I just verified this still present in NVDA 2020.4 + Chrome (93.0.4577.63) and Firefox 90.0.

I have something similar to the example in the previous comment but with `<input type="checkbox">` instead of buttons inside the lisitems. Both browse and focus mode only announce the checkboxes but not their position in the list.

**Comment by cppqtdev** [comment]
Hi Team 
I am also facing one issue with NVDA Screen Reader when the focus goes to the Listview in Qt /QML Application  
it is not reading 1 of 5 ,Let's total 5 item in the list and current item is 1 . then it should read name of the item and then read selected and then 1 of 5 or 1 of 5 item selected .


Note : When I tested this on Window Inbuilt Screen reader like narrator it read properly and also JAWS also read same behavior as Narrator screen reader .


Please support if possible .What we need to do ?

**Comment by jendemes** [comment]
+1 experiencing in 2022.4.0.27401 with Chrome 111.0.5563.65 and Firefox 111.0 .

The behavior is the same in both browsers: role="list" is announced and indicates the correct number of list items. Moving to an item with role="listitem" announces the inner content (a link wrapped in a div with role="listitem") but does not announce it is a list item.

It does respond to list controls and will move correctly through the list items with I/Shift+I, even if they are not announced as list items.
